example/lib/main.dart

```diff
@@ -141,6 +141,7 @@ Future<void> main() async {
   trufiCfg.url.twitter = globalCfg.getValue<String>("urlTwitter");
   trufiCfg.url.instagram = globalCfg.getValue<String>("urlInstagram");
   trufiCfg.url.share = globalCfg.getValue<String>("urlShare");
+  trufiCfg.url.isOtpGraphQL = globalCfg.getValue<bool>("isOtpGraphQL");
```

Comment:
instead a bool, I think is better if we just use a string with the type. e.g. `otpType = "GraphQl" | "default"`

Reply:
Or maybe create an enum? - Should we even use the GlobalCfg or the TrufiConfiguration signleton?

Reply:
even better, I was think it will come from the `config.json` file, but this is not the case.

Reply:
I guess at the moment it comes from the app_config.json file but we could add it to the TrufiConfiguration. 
https://github.com/trufi-association/trufi-core/blob/master/lib/trufi_configuration.dart#L31